mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/annotation/ComponentIdFieldTypeAnnotation.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.declaration.type.annotation;
+
+import org.mule.metadata.api.annotation.TypeAnnotation;
+import org.mule.metadata.api.model.ObjectFieldType;
+import org.mule.runtime.api.meta.model.parameter.ParameterModel;
+
+/**
+ * Marker annotation for a {@link ObjectFieldType} that represents a {@link ParameterModel ::isComponentId componentId}.
```

Comment:
Is it right that JavaDoc tag? `{@link ParameterModel ::isComponentId componentId}`. should not be {@link ParameterModel#isComponentId componentId}